### ISSUE TITLE
Update CSS

### ISSUE THREAD

**Issue by deejkelms** [rmrogers1/dizzydish3#1]
We should probably come up with a better system for CSS, It already is getting pretty and hard to follow. 

Here's a link with some different options:

https://blog.logrocket.com/styling-react-5-ways-style-react-apps/#sass-css-stylesheets-react

**Comment by deejkelms** [comment]
Looking into adding sass, heres a good article on file structure
https://dev.to/dostonnabotov/a-modern-sass-folder-structure-330f

**Comment by Mstanic05** [comment]
Can you assign this to me!
